modules/material/themes/material/mfa/other_mfas.php

```diff
@@ -29,23 +11,16 @@ function excludeSelf($others, $selfId) {
     </button>
     <ul class="mdl-menu mdl-js-menu mdl-menu--top-left" data-mdl-for="others">
         <?php
-        // TODO: remove this
         foreach ($otherOptions as $option) {
-            $type = $option['type'];
-
-            $callback = $option['callback'] ?? '/module.php/mfa/prompt-for-mfa.php?StateId='.htmlentities($this->data['stateId']).'&mfaId='.htmlentities($option['id']);
-
-            $image = 'mfa-' . $type . '.svg';
-            $altText = $this->t('{material:mfa:' . $type . '_icon}');
         ?>
-        <li class="mdl-menu__item" onclick="location.href = '<?= $callback ?>'">
+        <li class="mdl-menu__item" onclick="location.href = '<?= $option['callback'] ?>'">
             <span class="mdl-list__item-primary-content">
-                <img class="mdl-list__item-icon" src="<?= $image ?>" alt="<?= $altText ?>">
-
-                <?php
-                $label = empty($option['id']) ? 'help' : $type;
-                ?>
-                <?= $this->t('{material:mfa:use_' . $label . '}') ?>
+                <img
+                  class="mdl-list__item-icon"
+                  src="<?= $option['image'] ?>"
+                  alt="<?= $this->t('{material:mfa:' . $option['type'] . '_icon}') ?>"
+                >
+                <?= $this->t('{material:mfa:use_' . $option['label'] . '}') ?>
```

Comment:
Not the same logic as 46. Is there a reason?

Reply:
The logic was moved to modules/mfa/www/prompt-for-mfa.php line 115:

```php
$option['label'] = empty($option['id']) ? 'help' : $option['type'];
```

Brief aside: this is exemplary of the reason I wanted to move the modules all into ssp-base. Prior to that effort, these two files were in two different repositories. It was that much more difficult to synchronize the interface between the modules (e.g. mfa and material) with them in different repos.